Assets/Mirror/Tests/Editor/MessageBaseTests.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Mirror.Tests.MessageTests
 {
-    struct TestMessage : IMessageBase
+    struct TestMessage : INetworkMessage
```

Comment:
> Messages must be public

Do this need to be public? if not we need to clarify when message need to be public

Reply:
it is by default internal, which is fine.  

Basically, there will be a method in another class in this assembly that will access this structure (the generated reader and writer).  As long as it can access this struct,  then it is fine.  This means all types supported by Mirror need to be internal or public. 

We can definitely add something in the docs